app/SharedWebComponents/Components/VoiceDialog.razor.cs

```diff
@@ -55,7 +64,18 @@ private async Task GetVoicesAsync(bool isFromCallback = false)
 
     private void OnCancel() => Dialog.Close(DialogResult.Ok(_voicePreferences));
 
-    public void Dispose() => SpeechSynthesis.UnsubscribeFromVoicesChanged();
+    public void Dispose()
+    {
+        try
+        {
+            SpeechSynthesis.UnsubscribeFromVoicesChanged();
+        }
+        catch
+        {
+            // do nothing if the service does not support observation
```

Comment:
Same question as above... what's the concern this addresses?

Reply:
Added a much better comment.